app/dashboards/library_hour_dashboard.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
```

Comment:
I'm not sure we want this dashboard at all. No one should be editing hours here. I know it exists just to have a sync button. Maybe what we need is a completely overridden Dashboard page that just has a big sync button and lists the last time a sync was run (based on the date with the most recent update_date time?

Reply:
Not going to block this PR for that, but going to create a new issue based off this idea.